haskell-language-server.cabal

```diff
@@ -1763,19 +1765,17 @@ test-suite hls-semantic-tokens-plugin-tests
     , aeson
     , base
     , containers
+    , data-default
     , filepath
+    , ghcide                == 2.8.0.0
     , haskell-language-server:hls-semantic-tokens-plugin
-    , hls-test-utils              == 2.8.0.0
-    , hls-plugin-api
```

Comment:
I sorted these 2 dependency lists alphabetically, because I kept getting "unused package hls-plugin-api" warning, even though I deleted it. It turns out it was in the list twice :facepalm:

Reply:
Perhaps we should always format our .cabal files using a formatter? I know, such a suggestion is deeply abhorrent to most haskelleres, but now we have two great formatters, might as well use them :upside_down_face: 

I'll add it to the list of talking points for the next meeting.

Reply:
Count me in as a supporter. I cabal-fmt all my private project's cabal files.